torchvision/io/__init__.py

```diff
@@ -87,15 +91,24 @@ class VideoReader:
 
     def __init__(self, path, stream="video"):
         if not _has_video_opt():
-            raise RuntimeError("Not compiled with video_reader support")
+            raise RuntimeError(
+                "Not compiled with video_reader support, "
+                + "to enable video_reader support, please install "
+                + "ffmpeg (version 4.2 is currently supported) and"
+                + "build torchvision from source."
+            )
         self._c = torch.classes.torchvision.Video(path, stream)
 
     def __next__(self):
-        """Decodes and returns the next frame of the current stream
+        """Decodes and returns the next frame of the current stream.
+        At the moment, frames are encoded as a dict with mandatory
```

Comment:
Can we avoid saying "At the moment", and instead say that we return a dict with data and pts, with pts the timestamp in seconds?

I think it's better to avoid having in the documentation things that might indicate "at the moment". In general, the documentation is meant to represent the state of the code at the moment, so it's redundant and I'm not sure it reads great.

Reply:
Done